app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -213,7 +215,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.adminSignUp(body)
+        val actualResponse = apiService.adminSignUp(bearerToken, body)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver